### PR TITLE
Add the locale of the users browser to a slow request.

### PR DESCRIPTION
**Pull request by christ66**
@reviewbybees

### CODE REVIEW

**Comment by ydubreuil** [comment]
:bee:

**Comment by oleg-nenashev** [comment]
:bee:

File: src/main/java/com/cloudbees/jenkins/support/slowrequest/InflightRequest.java [hunk 1]
```diff
@@ -65,6 +70,7 @@
         referer = req.getHeader("Referer");
         String agentHeader = req.getHeader("User-Agent");
         userAgent = agentHeader != null ? UADetectorServiceFactory.getResourceModuleParser().parse(agentHeader) : null;
+        locale = req.getLocale().toString();
```
Comment (jglick) [inline src/main/java/com/cloudbees/jenkins/support/slowrequest/InflightRequest.java]: [OK]http://stackoverflow.com/a/28705163/12916)

**Comment by jglick** [comment]
:bee:

**Comment by aheritier** [comment]
:bee:

**Pull request merged**